README.md

```diff
@@ -396,6 +396,16 @@ application](deploy/common/pmem-app-cache.yaml) example.
 * A node is only chosen the first time a pod starts. After that it will always restart
   on that node, because that is where the persistent volume was created.
 
+Volume requests embedded in Pod spec are provisioned as ephemeral volumes. The volume request could use below fields as [`volumeAttributes`](https://kubernetes.io/docs/concepts/storage/volumes/#csi):
+
+|key|meaning|optional|default value|
+|---|-------|--------|-------------|
+|size|Size of the requested ephemeral volume|No||
+|nsmode|PMEM namespace mode of requested ephemeral volume|Yes|"fsdax"|
+|eraseAfter|Should erase volume content after use|Yes|"true"|
```

Comment:
"Should" or "will"? I think the meaning becomes more obvious when using a different verb in the description: "eraseAfter - clear all data after use and before deleting the volume"

Reply:
Replaced the description as suggested.